### PR TITLE
Add build spec for alinux gcc11

### PR DESCRIPTION
**Pull request by pshipton**

### CODE REVIEW

**Comment by pshipton** [comment]
jenkins compile alinux64gcc11 jdk19

**Comment by pshipton** [comment]
jenkins compile alinux64gcc11 jdk19

**Comment by pshipton** [comment]
jenkins compile alinux64gcc11 jdk19

**Comment by pshipton** [comment]
jenkins compile alinux64gcc11 jdk19

**Comment by pshipton** [comment]
jenkins compile alinux64gcc11 jdk19

**Comment by pshipton** [comment]
Old build using `:latest` https://openj9-jenkins.osuosl.org/job/Build_JDK19_aarch64_linux_gcc11_Personal/4

**Comment by pshipton** [comment]
jenkins compile alinux64gcc11 jdk19

**Comment by pshipton** [comment]
Whether or not the compile succeeds, the spec is working.

File: buildenv/jenkins/variables/defaults.yml [hunk 1]
```diff
@@ -566,3 +566,10 @@ s390x_linux_gcc11:
   build_env:
     cmd:
       all: 'source /home/jenkins/set_gcc_11.2.0_env'
+#========================================#
+# Linux Aarch64 64bits compiled with GCC 11
+#========================================#
+aarch64_linux_gcc11:
+  extends: ['aarch64_linux']
+  build_env:
+    vars: 'CC=/usr/local/gcc11/bin/gcc-11.2 CXX=/usr/local/gcc11/bin/g++-11.2 LD_LIBRARY_PATH=/usr/local/gcc11/lib64:/usr/local/gcc11/lib'
```
Comment (keithc-ca) [inline buildenv/jenkins/variables/defaults.yml:575]: Why doesn't this source the script like other platforms:
```
x86-64_linux_gcc11:
  extends: ['x86-64_linux']
  build_env:
    cmd:
      all: 'source /home/jenkins/set_gcc_11.2.0_env'
```

**Review by keithc-ca — APPROVED** [review]
Even though we know builds currently fail, this will ease the work to fix that.

**Pull request merged**